projects/common/src/time/time-range.service.ts

```diff
@@ -93,6 +92,21 @@ export class TimeRangeService {
     );
   }
 
+  private initializeTimeRange(): void {
+    concat(this.getInitialTimeRange(), this.getPageTimeRangeChanges()).subscribe(timeRange => {
```

Comment:
one option discussed offline - the two branches of the subscribe basically handle the respective emissions of these two observables. We could do it as two separate subscriptions and it would probably be easier to understand.